rskj-core/src/main/java/org/ethereum/vm/program/invoke/ProgramInvokeImpl.java

```diff
@@ -302,6 +303,11 @@ public int getCallDeep() {
         return this.callDeep;
     }
 
+    @Override
+    public int hashCode() {
+        return address.hashCode();
```

Comment:
This implementation is not correct. If we're not using `equals`, I would suggest removing both `equals` and `hashCode`.